### PR TITLE
Add about the authors information to XHTML export routine

### PR DESCRIPTION
**Pull request by richard015ar**
Related issue: https://github.com/pressbooks/pressbooks/issues/2347

This PR adds the **About the Authors** section at the end of each chapter for XHTML and EPUB (using [the pb-books PR](https://github.com/pressbooks/pressbooks-book/pull/853)) exports, including PDFs through a new method that adds the HTML at the end of the string of each chapter .

This PR requires pressbooks-book change, see: https://github.com/pressbooks/pressbooks-book/pull/853

### Test
- For a random book, assign different authors to a different chapters. Make sure those authors are book's contributor in the Book Info section.
- Make sure Appearance > Theme Options > About the authors options is enabled.
- To test it better, you may add picture, url, description and/or other fields to the contributors in Book Info > Contributors section.
- Export the book to EPUB, PDF and XHTML and make sure the authors are added at the end of those chapter with authors.

### CODE REVIEW

File: inc/modules/export/namespace.php [hunk 1]
```diff
@@ -279,3 +281,47 @@ function update_pins() {
...
+	if ( empty( $chapter_contributors ) ) {
+		return '';
+	}
+	$title = sprintf( _n( '%s Author', '%s Authors', count( $chapter_contributors ), 'pressbooks' ), 'About the' );
+	$print = '<div class="contributors">';
+	$print .= "<h3 class=\"about-authors\">{$title}</h3>";
+	foreach ( $chapter_contributors as $contributor ) {
+		$print .= '<div class="contributor_name_and_links">';
+		if ( $contributor['contributor_picture'] ) {
+			$print .= "<img class=\"contributor_profile_picture\" src=\"{$contributor['contributor_picture']}\" />";
```
Comment (fdalcin) [inline inc/modules/export/namespace.php:303]: EPUB validation is complaining about missing alt tags here

**Review by fdalcin — CHANGES_REQUESTED** [review]
Hey @richard015ar LGTM. I just noticed that when exporting the EPUB 2.01, it gives a warning about validation errors.

image

Logs are here:
image

Other minor things I believe are related to the styling portion of this PR.

I feel like the authors printed below the chapter name could have a smaller size, not much though.
image

For the links, I have the same feeling that we could reduce the font size.
image

File: inc/modules/export/namespace.php (hunk 1 above)
Comment (SteelWagstaff) [reply inc/modules/export/namespace.php:303]: For alt text, we should attempt to include/display the image's alt text and fall back to `alt=""` if none has been set for the image in question. https://webaim.org/techniques/alttext/#decorative

**Comment by richard015ar** [comment]
@fdalcin thank you for reviewing it.
I added the alt attribute for the img tag, now EPUB validation is not throwing any issues.
Take in consideration (which is not the previous case you mentioned in the PR comment) where some validation issues are thrown, it will depend of the book structure itself.

Regarding the authors in the index of the book is not something was added in this PR, it is another feature.

For the links size, as you mentioned to me, it is for [the related PR](pressbooks/pressbooks-book#853) but looks like a smaller size would looks nicer, what do you think @SteelWagstaff? Let's discuss it in the review meeting we have today.

**Comment by SteelWagstaff** [comment]
> For the links size, as you mentioned to me, it is for [the related PR](pressbooks/pressbooks-book#853) but looks like a smaller size would looks nicer, what do you think @SteelWagstaff? Let's discuss it in the review meeting we have today.

Link size could be adjusted in the styling PR, it's true, but I think it's outside the scope of _this_ particular task. I don't think it should be a blocker for this pressbooks PR.

**Review by fdalcin — APPROVED** [review]
Thanks @richard015ar, LGTM. I'll wait to merge until the styling PR is approved. 

Agreed, minor styling changes shouldn't be blocker for this.

**Pull request merged**